tear-down-azure.sh

```diff
@@ -6,4 +6,6 @@ then
   exit
 fi
 
-helm delete bc -n $1
\ No newline at end of file
+helm delete bc -n $1
+
+kubectl delete ns $1
```

Comment:
I'd argue that for readability's sake we should use the variable that has $1 assigned to it

So instead of:
```sh
helm delete bc -n $1

kubectl delete ns $1
```

Like so:

```sh
helm delete bc -n $namespace

kubectl delete ns $namespace
```

what do you think?

Reply:
agreed